<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_sua_giang_vien</class>
 <widget class="QWidget" name="man_hinh_sua_giang_vien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string>Tên giảng viên: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ten_giang_vien">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(0, 112, 192);
background-color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(0, 112, 192);
background-color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="thoat">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(0,0,0);
background-color: rgb(255, 85, 0);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>THOÁT</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/newPrefix/cnpm/32_n.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="sua">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(0,0,0);
background-color: rgb(85, 170, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>SỬA</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/334_n.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string>Tên tài khoản:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string>Mật khẩu:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ten_tai_khoan">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(0, 112, 192);
background-color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mat_khau">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(0, 112, 192);
background-color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string>Tình trạng:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tinh_trang">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(0, 112, 192);
background-color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>label</zorder>
  <zorder>ten_giang_vien</zorder>
  <zorder>thoat</zorder>
  <zorder>sua</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>ten_tai_khoan</zorder>
  <zorder>mat_khau</zorder>
  <zorder>label_4</zorder>
  <zorder>tinh_trang</zorder>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
